beast/physicsmodel/helpers/hdfstore.py

```diff
@@ -1,27 +1,30 @@
 """ HDFStore
 High level interface to PyTables for reading and writing structures to disk
 Could work either as a (key, value) storage or as a classic tables.file.File object.
-
-    Example usage
-    >>> import numpy as np
-        #make a store
-        with HDFStore('tmp.hd5', mode='w') as hd:
-            #make some variables
-            d = {}
-            d['a'] = np.arange(10, dtype=float)
-            d['b'] = np.arange(10, dtype='int')
-            c = np.random.normal(0, 1, (10, 10))
-            d['c'] = c
-            #put values into the store
-            hd['/subdir/table'] = d
-            hd['/subdir1/array'] = c
-            #check values
-            print hd.keys()
-            print hd['subdir/table']
-            print hd['subdir1/array']
-            hd.remove_node('/subdir1', recursive=True)
-
 """
+# this code block fails when 'python setup.py test' is run
```

Comment:
This is the error that I get with this code.  Basically, it is running the code on the >>> lines and expecting the lines afterwards to be the output.   As this is not happening, it fails.  I thought the best solution would be to comment it out for now and come back and "fix" it.  Not quite sure what the fix is yet.  Better to get the travis ci working right now so that we have at least basic testing of the code.  Lots of documentation fixing needed anyway.

_________________________________________ [doctest] beast.physicsmodel.helpers.hdfstore __________________________________________
001 """ HDFStore
002 High level interface to PyTables for reading and writing structures to disk
003 Could work either as a (key, value) storage or as a classic tables.file.File object.
004     >>> import numpy as np
Expected:
        #make a store
        with HDFStore('tmp.hd5', mode='w') as hd:
            #make some variables
            d = {}
            d['a'] = np.arange(10, dtype=float)
            d['b'] = np.arange(10, dtype='int')
            c = np.random.normal(0, 1, (10, 10))
            d['c'] = c
            #put values into the store
            hd['/subdir/table'] = d
            hd['/subdir1/array'] = c
            #check values
            print hd.keys()
            print hd['subdir/table']
            print hd['subdir1/array']
            hd.remove_node('/subdir1', recursive=True)
Got nothing

beast/physicsmodel/helpers/hdfstore.py:4: DocTestFailure